static/mass-verify.js

```diff
@@ -0,0 +1,257 @@
+            notFound.push(csvResults[i+1])
+        }
+    }
+    createTable();
+}
+function createTable(){
+// Create a table
+var table = document.createElement('table');
+
+// Create table header
+var thead = document.createElement('thead');
+var headerRow = document.createElement('tr');
+['Member ID', 'Name', 'Email', 'Discord', 'Status'].forEach(headerText => {
+    var th = document.createElement('th');
+    th.textContent = headerText;
+    headerRow.appendChild(th);
+});
+thead.appendChild(headerRow);
+table.appendChild(thead);
+
+// Create table body
+var tbody = document.createElement('tbody');
+results.forEach(user => {
+    var row = document.createElement('tr');
+    ['id', 'name', 'email', 'discord.username', 'status'].forEach(key => {
+        var cell = document.createElement('td');
+        var value = user;
+        var keys = key.split('.');
+        for (var i = 0; i < keys.length; i++) {
+            value = value[keys[i]];
+        }
+        if (keys.includes('email')) {
+            cell.classList.add('email');
+        }
+        if (keys.includes('id')) {
+            cell.classList.add('id-truncate');
+            cell.title = value;
+        }
+        cell.textContent = value;
+        row.appendChild(cell);
+    });
+    tbody.appendChild(row);
+});
+table.appendChild(tbody);
+// Append the table to the body (or any other container element)
+var container = document.getElementById('userTable');
+container.appendChild(table);
+
+// Create table for notFound data
+var notFoundTable = document.createElement('table');
+// Create table header
+var notFoundThead = document.createElement('thead');
+notFoundTable.appendChild(notFoundThead);
+// Create table body
+var notFoundTbody = document.createElement('tbody');
+notFound.forEach(data => {
+    var row = document.createElement('tr');
+    var csvDataCell = document.createElement('td');
+    var statusCell = document.createElement('td');
+    data.split(',').forEach(cellData => {
+        var cell = document.createElement('td');
+        cell.textContent = cellData;
+        row.appendChild(cell);
+    });
+    notFoundTbody.appendChild(row);
+});
+notFoundTable.appendChild(notFoundTbody);
+
+// Append the notFound table to the body (or any other container element)
+var notFoundContainer = document.getElementById('notFoundTable');
+notFoundContainer.appendChild(notFoundTable);
+
+document.getElementById("results").style.display = "block";
+
+}
+
+function getColumnByTitle(csvData, title) {
+    const lines = csvData.split('\n');
+    const columnTitles = lines[0].split(',');
+    const columnIndex = columnTitles.indexOf(title);
+    const columnData = [];
+    
+    for (let i = 1; i < lines.length; i++) {
+        const rowData = lines[i].split(',');
+        if (rowData.length > columnIndex) {
+            columnData.push(rowData[columnIndex]);
+        }
+    }
+    
+    return columnData;
+}
+
+
+function downloadCSV() {
```

Comment:
You can use the HTML5 Download attribute instead of this solution now: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download